style.css

```diff
@@ -0,0 +1,1498 @@
+  background-repeat: no-repeat;
+}
+
+.card-3-content-8 {
+  margin-top: 16px;
+  padding: 12px;
+  height: 276px;
+  width: 100%;
+  background-image: url("https://img-cdn.tnwcdn.com/image?fit=1280%2C720&url=https%3A%2F%2Fcdn0.tnwcdn.com%2Fwp-content%2Fblogs.dir%2F1%2Ffiles%2F2020%2F07%2FShift-launch-cars-evs-future-polestar-electric-cars-driving-mobility.png&signature=ae628edcb255606b8d38e0e7a9a34998"), linear-gradient(transparent, rgba(0, 0, 0) 98%);
+  background-blend-mode: overlay;
+  background-position: top;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.card-3-content-9 {
+  margin-top: 16px;
+  padding: 12px;
+  height: 276px;
+  width: 100%;
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/08/Mulan-796x417.jpg"), linear-gradient(transparent, rgba(0, 0, 0) 98%);
+  background-blend-mode: overlay;
+  background-position: top;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.card-3-bottom {
+  background-color: rgba(255, 68, 34, 0.08);
+  color: #f42;
+  display: flex;
+  padding-left: 14px;
+}
+
+.card-3-bottom :first-child {
+  padding: 7px 0 1px;
+}
+
+.card-4-discount h4 {
+  transform: rotate(45deg);
+  font-size: 14px;
+  position: absolute;
+  top: 20px;
+  right: 0;
+}
+
+.card-3-bottom div h4 {
+  font-family: "ARS Maquette-bold", serif;
+  font-size: 14px;
+}
+
+.card-3 li::before {
+  content: "\2022";
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-top: 6px;
+}
+
+.grid-container-2 {
+  grid-template-columns: repeat(3, 1fr);
+  width: 100%;
+  grid-gap: 56px 24px;
+  margin-top: 113px;
+}
+
+.latest-deals {
+  width: 100%;
+  margin-top: 60px;
+}
+
+.latest-deals-container {
+  grid-template-columns: repeat(4, 1fr);
+  width: 100%;
+  grid-gap: 24px;
+  margin-top: 12px;
+  padding-bottom: 88px;
+}
+
+.card-4 {
+  width: 100%;
+}
+
+.card-4-discount-container {
+  position: relative;
+}
+
+.card-4 img {
+  width: 100%;
+  height: 201px;
+  object-fit: cover;
+}
+
+.border-right-special {
+  border-right: none;
+}
+
+.border {
+  border-right: 1px solid #f42;
+  border-left: 1px solid #f42;
+}
+
+.card-4-discount {
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 80px;
+  height: 80px;
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/deals-ribbon.svg");
+  background-repeat: no-repeat;
+}
+
+/* Footer Section */
+
+.footer-links {
+  width: 100%;
+  height: 161px;
+  padding: 45px;
+  background-color: #131313;
+  color: #555;
+}
+
+.footer-links-social {
+  margin-top: -20px;
+  display: flex;
+  list-style: none;
+  justify-content: center;
+}
+
+.footer-links-social svg {
+  filter: invert(1);
+}
+
+.footer-links-social svg:hover {
+  border: none;
+}
+
+.footer-links-menu {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+  margin: 20px 6px;
+}
+
+.footer-copyright {
+  display: flex;
+  justify-content: center;
```

Comment:
## Using the Same Rule Repetitively[1]
- You already have a display flex property in .flex class, please use it
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file

**Example:**


**Html file**

```HTML
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_